### ISSUE TITLE
Edit button is placed ideally only for articles you are likely to read through all of and then need to modify

### ISSUE THREAD

**Issue by fusion2004** [orientation/orientation#180]
We have a few long articles where only the very top of it is generally read or modified, and it ends up being a long scroll to the bottom to get to the edit button.

Some ideas:
- Place a second Edit button in or move the current one to the right sidebar.
- Make the Edit/Archive/Delete buttons float towards the bottom of the screen.

@johndjameson Thoughts?

**Comment by olivierlacan** [comment]
This has been an issue for me on very long articles too.

**Comment by johndjameson** [comment]
Agreed. We'll need to give the edit button some more priority on the article view page.

In the meantime, you can edit an article by pressing `e` on the keyboard.
